CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributor Onboarding
+
+🚧 This document is still a WIP 🚧 
+
+## General
+
+ - The goal of this guide is to help you start contributing to Durable Functions
+
+## Pre-reqs
+
+ - OS
+    - Windows 10 (suggested)
+    - Mac OS X/Linux
+ - Language runtimes
+    - .NET Core 2.0
+ - Editor
+    - Visual Studio 2017 (recommended)
+    - VS Code 
+ - Misc tools (suggested)
+    - git - source control
```

Comment:
The git prerequisite is probably well-understood since this is a GitHub repo. I think we can leave this out of the list.

Reply:
Good point. Thank you!